site/content/docs/5.0/examples/sidebars/index.html

```diff
@@ -0,0 +1,387 @@
+---
```

Comment:
need to check orange brand

Reply:
will be adressed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/380